manifests/init.pp

```diff
@@ -129,6 +130,9 @@
   }
 
   # validate params
+  validate_integer($vas_conf_vasypd_update_interval)
+  $vas_conf_vasypd_update_interval_integer = 0 + $vas_conf_vasypd_update_interval # str2integer
```

Comment:
Ok, I'll change it. Though the 0 + trick is actually the documented method [1]. :-)

[1] [https://docs.puppet.com/puppet/4.9/lang_data_number.html#converting-strings-to-numbers](url)